src/app/pages/landing.page.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Component } from '@angular/core';
+
+@Component({
+  styles: [
+    `.orange-accent { height: 2px; background-color: #ff7e29; }`
+  ],
+  template: `
+    <h2 cds-text="display" cds-layout="m-t:xl">Clarity Core Adoption Guide</h2>
+
+    <div cds-layout="grid cols:12 cols@sm:6 m-t:xl">
+      <p cds-text="section">
+          A simple, intuitive guide for helping teams adopt and migrate from Clarity Angular to Clarity Core.
+      </p>
+    </div>
+
+    <section cds-layout="container:xl">
+      <div cds-layout="grid cols@sm:12 cols@md:6 gap:xl align:vertical-center" class="home-section section-column-wrapper alternated">
+      <div cds-layout="vertical" cds-layout="display:block display@md:none">
+          <img src="/assets/img_build_framework.svg" alt="A hand holding icons for Angualr, Vue and React web frameworks." cds-layout="fill" />
```

Comment:
`alt="A hand holding icons for Angular,` not `alt="A hand holding icons for Angualr,`

Reply:
Ha that was copied from the main website...